_includes/footer.html

```diff
@@ -5,7 +5,7 @@
         </nav>
         <p>
           If you have questions or issues, it is always best to consult a legal professional.<br>
-          This site is licensed under the <a href="http://creativecommons.org/licenses/by/3.0/">
+          The content of this site is licensed under the <a href="http://creativecommons.org/licenses/by/3.0/">
```

Comment:
I would say that "content" encapsulates images and theming assets, but it doesn't include ops things like the way the the content is provided (the server and what not) which I would normally consider under the umbrella of "this site". My $0.02.